lib/biokbase/catalog/Impl.py

```diff
@@ -20,9 +20,9 @@ class Catalog:
     # state. A method could easily clobber the state set by another while
     # the latter method is running.
     ######################################### noqa
-    VERSION = "2.1.2"
-    GIT_URL = "https://github.com/JamesJeffryes/catalog.git"
-    GIT_COMMIT_HASH = "82480c4f938d5d1d5a85668a0ebaf1f27bf793f3"
+    VERSION = "0.0.1"
```

Comment:
The version is probably not 0.0.1

Reply:
it's not registered in the catalog (so meta) so the version in impl is irrelevant